src/routes/docs/examples/Sparkline/+page.svelte

```diff
@@ -0,0 +1,104 @@
+<script lang="ts">
+  import { scaleTime } from 'd3-scale';
+  import { format } from 'date-fns';
+
+  import Chart, { Svg } from '$lib/components/Chart.svelte';
+  import Highlight from '$lib/components/Highlight.svelte';
+
+  import Spline from '$lib/components/Spline.svelte';
+  import Tooltip from '$lib/components/Tooltip.svelte';
+  import TooltipItem from '$lib/components/TooltipItem.svelte';
+
+  import Preview from '$lib/docs/Preview.svelte';
+  import { createDateSeries } from '$lib/utils/genData';
+
+  const data = createDateSeries({ count: 50, min: 50, max: 100, value: 'integer' });
+
+</script>
+
+<h1>Examples</h1>
+
+<h2>Basic</h2>
+<Preview>
+  <div>
+    <div class="w-[125px] h-[18px]">
+      <Chart
+        {data}
+        x="date"
+        xScale={scaleTime()}
+        y="value"
+        yDomain={[null, null]}
```

Comment:
Specifying the `yDomain` shouldn't be needed here as `[null,null]` is the same as the default of calculating the min/max, unless I'm overlooking something.

Reply:
You're correct. yDomain in not needed here.